test/Parse/generic_disambiguation.swift

```diff
@@ -74,3 +74,15 @@ A<B?>(x: 0) // parses as type // expected-warning{{unused}}
 _ = a < b ? c : d
 
 A<(B) throws -> D>(x: 0) // expected-warning{{unused}}
+
+struct G<T,U> {}
+_ = A<A<A<A<A<B>>>>>() // not enough closing brackets
+_ = A<A<A<A<A<B>> >>>() // space in closing brackets
```

Comment:
> We check for Perforce-style conflict markers too, so you only need 4.

I don't understand. Who checks? Are you arguing that I change the code to warn starting at `>>>>`?